pybamm/parameters/parameter_sets.py

```diff
@@ -113,6 +113,18 @@
     "citation": "Chen2020",
 }
 
+Chen2020_plating = {
+    "chemistry": "lithium-ion",
+    "cell": "LGM50_Chen2020",
+    "anode": "graphite_Chen2020_plating",
+    "separator": "separator_Chen2020",
+    "cathode": "nmc_Chen2020",
+    "electrolyte": "lipf6_Nyman2008",
+    "experiment": "1C_discharge_from_full_Chen2020",
+    "sei": "example",
+    "citation": "Chen2020",
```

Comment:
Apart from `Chen2020` you should also add the reference for the plating parameters.